editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -584,9 +584,12 @@ function useSelectOrLiveModeSelectAndHover(
 } {
   const dispatch = useEditorState((store) => store.dispatch, 'useSelectAndHover dispatch')
   const selectedViewsRef = useRefEditorState((store) => store.editor.selectedViews)
+  const isZoomMode = useEditorState(
+    (store) => store.editor.keysPressed['z'] ?? false,
```

Comment:
Is there any risk that this catches `z` pressed not on the canvas, or in circumstances not related to wanting to use the mouse to zoom in?